man/kernel-command-line.xml

```diff
@@ -382,6 +382,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>systemd.watchdog_device=</varname></term>
+
+        <listitem>
+          <para>Overwrites the watchdog device path <varname>WatchdogDevice=</varname> at boot. For details, see
```

Comment:
I'd drop "at boot" here. We have various options which apply only at boot, and this one applies even after boot, and in particular during shutdown.